stdlib/Markdown/docs/src/index.md

```diff
@@ -67,6 +69,12 @@ A paragraph containing some ``\LaTeX`` markup.
     ```
     @doc raw"``\LaTeX`` syntax in a docstring." functionname
     ```
+    
+!!! note
+    Inline ``\LaTeX`` may also be created within a set of single of `$` characters, though this is 
+    depreciated. The ``` `` ``` syntax is preferred as `$` is also used for string interpolation -
+    the parser will interpret an accidentally unmatched `$` as an intented string interpolation 
```

Comment:
If it's inline, it can't really be indented? I think this sentence should be rephrased (see also the note below).

Reply:
I think "intented" was meant to be "intended" not as "indented", which makes some sense, but still should be changed.

But is this all even true? I thought `$` is always parsed as string interpolation because Julia does this before any documentation code ever sees the docstring?

Maybe that's a feature of the (undocumented) `md_str` ? In any case, it seems misleading to discuss this here. Maybe better to say something to the effect of "Don't use `$` as it is interpreted as string interpolation; if you really want to use it, use a `raw` string" ?